File: 511_library/Video.cpp
#include "Video.h"

#include <iostream>
#include <conio.h>



Video::  Video (string call_number4,string title4,	string subjects4,
	string description4, string distributor4 , string notes4 , 
	string series4,string label4 ):Media(call_number4, title4,subjects4, notes4)
{
	//call_number = call_number4;
	//title= title4;
	//subjects = subjects4 ;
	description = description4 ;
	distributor = distributor4;
	//notes = notes4;
	series = series4;
	label = label4;
}

void Video::display() const
{
	cout<<"VIDEO"<<endl;
	Media::display();
	cout<<"Description     :"<<description<<endl;
	cout<<"Distributor    :"<<distributor<<endl;
	cout<<"Series   :"<<series<<endl;
	cout<<"label    :"<<label<<endl;
}


bool Video::compare_other(const string& ss) 
{
	//int i = -1;
	if (std::string::npos != description.find(ss)||std::string::npos != notes.find(ss)|| std::string::npos != distributor.find(ss))
	{
		return true;

	}
	else
		return false;
	
	/*if (std::string::npos != distributor.find(ss))
	{
		if (i >= 0)
			return true;
		else
			return false;
	}

	if (std::string::npos != notes.find(ss))
	{
		if (i >= 0)
			return true;
		else
			return false;
	}*/
}

File: Library_Python/Video.py
from Media import Media
class Video(Media):
    def __init__(self,callNumber1,title1,subjects1,description1,distribution1,notes1,series1,label1):
        super(Video,self).__init__(callNumber1,title1,subjects1,notes1)
        self.description=description1
        self.distribution = distribution1
        self.series = series1;
        self.label = label1;

    def display(self):
        print("Results")
        print("VIDEO")
        super(Video,self).display()
        print(self.description)
        print(self.distribution)
        print(self.series)
        print(self.label)

    def compare_other(self,string_others):
        if (self.description.find(string_others) != -1 or self.notes.find(string_others) != -1 or self.description.find(string_others)!= -1):
            return True
        return False
        
